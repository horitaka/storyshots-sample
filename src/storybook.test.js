import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyNameRegex: /^((?!.*?Excluedsnapshot).)*$/
});
