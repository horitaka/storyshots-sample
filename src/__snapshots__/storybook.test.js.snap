// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots MaterialButton Button 1`] = `
<div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Button
    </span>
  </button>
</div>
`;

exports[`Storyshots StyledButton Button 1`] = `
<button
  className="sc-AxjAm gxygnu"
>
  Button
</button>
`;
